docs/next/guides/building-and-testing/modules.md

```diff
@@ -9,251 +9,44 @@ tags:
 
 # Modules
 
-To reduce the size of your app, you can use Handsontable by importing individual [modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules).
+To reduce the size of your app, use Handsontable modules.
 
 [[toc]]
 
 ## About modules
 
-The full bundle of Handsontable consists of multiple built-in modules such as plugins, renderers, editors, validators, and cell-types. It also includes the whole rendering functionality and advanced data management. This is a compact tool with plenty of options available.
+Modules let you pick and choose only those parts of Handsontable that you actually need. This can significantly reduce the size of your app.
```

Comment:
@warpech - if by "old text" you mean this:

<img width="823" alt="1" src="https://user-images.githubusercontent.com/29247866/150368477-ccc053f5-12a2-4be2-8de8-a841081ccafd.png">

Then I'm not sure I agree - perhaps, it could be good material for a blog post, but not for technical documentation.

IMO this page should:
- Give a **brief** explanation of what Handsontable's modules are and why you should use them  ([conceptual information](https://github.com/handsontable/handsontable/pull/9059#discussion_r788794087)) (without trying to explain what modules are in general and how great that feature is, because that's off-topic – instead, point the user to other resources)
- Present a clear list of all available modules ([reference information](https://github.com/handsontable/handsontable/pull/9059#discussion_r788794087))
- Guide the user through the process of importing, registering and using modules of all available types ([task information](https://github.com/handsontable/handsontable/pull/9059#discussion_r788794087))

If we want to expand the **About modules** section but stick to actual documentation content, we could do it more or less like this (fixing the distribution/bundle/package discrepancy on the way):

# About modules

Handsontable is a comprehensive tool with a broad range of features. If you don't need all of them, you can pick only those parts that you actually use, and get rid of the rest (e.g. unnecessary translations). This approach reduces Handsontable's impact on the overall size of your app.

To make this possible, Handsontable is divided into [modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules). The [full distribution/bundle/package](https://handsontable.com/docs/packages/#full-distribution-recommended) of Handsontable contains all the available modules, but you can decide to import only the ones that you actually use.

Learn more about JavaScript modules:
- MDN Web Docs: [JavaScript modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)
- Handsontable blog: [Modularizing to improve the developer experience](https://handsontable.com/blog/articles/2021/2/modularizing-to-improve-the-developer-experience)
- Handsontable blog: [Modularization for React, Angular, and Vue](https://handsontable.com/blog/articles/2021/11/handsontable-11.0.0-modularization-for-react-angular-and-vue)

Reply:
> @warpech - if by "old text" you mean this:

By "old text" I only meant the first paragraph. The old:

> The full bundle of Handsontable consists of multiple built-in modules such as plugins, renderers, editors, validators, and cell-types. It also includes the whole rendering functionality and advanced data management. This is a compact tool with plenty of options available.

Was better than the then-current:

> Modules let you pick and choose only those parts of Handsontable that you actually need. This can significantly reduce the size of your app.

The current version in ace44850b1fa0c41bbfa7b7781dc86fe09b737ad solved by problem.